### DISCUSSION TITLE
warning before making changes to the main code during execution, similar to Notepad++

### DISCUSSION THREAD

**Discussion by 360nama** [microsoft/vscode-discussions#2397] [Extension Development QnA] [answered]
"I want Visual Studio Code to show a ![ekhtar](https://github.com/user-attachments/assets/875339fe-c001-400c-83a0-b421ff7136dc). This is the only major drawback that prevents me from working with Visual Studio Code. I am currently working on web development on a local server with WAMP, and I would like to have a warning before any changes. Several times, code was added automatically, and my project was ruined."

**Comment by alefragnani** [comment, marked as answer]
Hi @gilanweb ,

VS Code does not have a setting to customize this behavior. There is however, an open issue https://github.com/microsoft/vscode/issues/23107 asking something like this, so I suggest you to take a look and add your comments/scenario if necessary. It is in the backlog, but no ETA.

BTW, this discussion repo is for extension development only, so, for questions about VS Code usage, I suggest you to take a look on [Stack Overflow](https://stackoverflow.com/questions/tagged/vscode).

Hope this helps
